src/main/resources/handlebars/typescript-angular/api.service.mustache

```diff
@@ -337,3 +337,4 @@ export class {{classname}} {
 
 {{/operation}}
 {{/operations}}
+}
```

Comment:
i think this is not the right place to put the curly brace. it should be placed between those closing tags/elements.
```
{{/operation}}
}
{{/operations}}
```